plugin/dev/src/test/java/org/opencds/cqf/ruler/plugin/dev/TesterUtilities.java

```diff
@@ -0,0 +1,8 @@
+package org.opencds.cqf.ruler.plugin.dev;
```

Comment:
Use / move this to the test-utility plugin.

Reply:
I removed it like the other unnecessary Utilities